root/etc/cont-init.d/11-folders

```diff
@@ -4,7 +4,7 @@
 mkdir -p \
     /config/{keys,php,www} \
     /config/log/{nginx,php} \
-    /config/nginx/{http-confs,location-confs,server-confs,site-confs} \
```

Comment:
It's worth noting that the folder structure rollback is the most visually obvious change happening here (for devs and end users to see). Most of the other changes are inside the image and confs and won't stick out as blatantly.